add explicit targetless (#169)
metalbear-co/mirrord.dev: content/docs/reference/targets/index.md

on your machine. So if you're using a management program that exposes a web interface, you can have it listen for
connections on `localhost`, and connect to remote services in the cluster.

> In order to run targetless, you can either specify explicitly in the configuration file, or just don't set it in it.

Reviewer: ```suggestion
In order to run targetless and are using a mirrord configuration file, you can either specify it explicitly or just leave the `target` key out completely.
```

> In order to run targetless and are using a mirrord configuration file, you can either specify it explicitly or just leave the `target` key out completely.

Reviewer: ```suggestion
If you're using a mirrord configuration file and want to run targetless, you can either leave the `target` key out completely or specify it explicitly.
```

> If you want to specify it explicitly (might be useful in Java/Kotlin where the target selection dialog doesn't work):

Reviewer: ```suggestion
If you want to specify it explicitly (which might be useful when running Java/Kotlin projects in IntelliJ IDEs, in which case the target selection dialog doesn't work):
```

> If you want to specify it explicitly (which might be useful when running Java/Kotlin projects in IntelliJ IDEs, in which case the target selection dialog doesn't work):

Reviewer: ```suggestion
Note that if you want to skip the target dialog in the IDE plugins, you have to specify it explicitly. You can do it with the following configuration:
```
